client/app/account/user/credentials/user-credentials.service.js

```diff
@@ -0,0 +1,43 @@
+angular.module("UserAccount.services").service("UserAccount.services.credentials", [
```

Comment:
Feel free to write this as an entire ES6 class.

Prefer naming param as `credentialId` instead of `credId`.